src/auth/casl-ability.factory.ts

```diff
@@ -126,6 +129,9 @@ export class CaslAbilityFactory {
       can(Permissions.Update, 'Consultation')
       can(Permissions.Delete, 'Consultation')
       can(Permissions.DownloadFile, 'Consultation')
+      cannot(Permissions.JoinConsultation, 'Consultation')
```

Comment:
`cannot` is not actually needed here. CASL won't let you access the resource unless it was explicitly specified with a `can`. The only use of `cannot` is to override a previous `can`. This `cannot` even prevents admins from joining their own consultations, which isn't really a problem, but still kinda unexpected behaviour